### PR TITLE
1565  - fix spacing in export modal

### PR DESCRIPTION
**Pull request by rowasc**
This pull request makes the following changes:
- Corrects a space in the modal for export feedback, which I broke when doing adjustments around 2044/2045/1565

Testing checklist:
- [ ] The modal should look like this; 

screen shot 2017-09-28 at 11 14 58 am
- [ ] The modal should not look like this: 

screen shot 2017-09-28 at 11 19 18 am

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by rowasc** [comment]
@crcommons do you have some time to do a CR of this? Thanks!

File: app/common/locales/en.json [hunk 1]
```diff
@@ -1129,7 +1129,7 @@
             "sent_to" : "Message sent to {{contact}}"
         },
         "export" : {
-            "in_progress" : "<p><h3>Your CSV export is in progress...</h3>You can navigate away from this window, we'll let you know when your download is complete</p><br>",
+            "in_progress" : "<h3>Your CSV export is in progress...</h3>You can navigate away from this window, we'll let you know when your download is complete.</p><br>",
```
Comment (rowasc) [inline app/common/locales/en.json]: ok this is wrong. Sending a fix.

**Comment by rowasc** [comment]
@crcommons perfect, thank yoU!

**Comment by crcommons** [comment]
LGTM!

**Pull request merged**